.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,27 @@
+name: Lint and Test Helm-Chart
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Are you sure this will suffice as chart-testing has to detect chart changes. Checking the [chart-testing-action repo](https://github.com/helm/chart-testing-action) the example even fetches the history with

```shell
      - name: Fetch history
        run: git fetch --prune --unshallow
```

Reply:
The whole history is fetched if `fetch-depth` is set to `0`. (https://github.com/actions/checkout#usage) But I can change this if you want.